src/Spectre.Algorithms/Methods/Segmentation.cs

```diff
@@ -66,7 +66,7 @@ public DivikResult Divik(IDataset dataset, DivikOptions options)
 
 			var varargin = options.ToVarargin();
 			var tmp = _segmentation.divik(numberOfOutputArgs, dataset.GetRawIntensities(), coordinates, varargin);
-			var result = new DivikResult(tmp);
+			var result = new DivikResult(tmp[1]);
```

Comment:
Please place the same explanation in code (no one in future will check this pull request) ;)